src/Service/CaptureServiceImpl.cpp

```diff
@@ -200,6 +201,13 @@ static void StopInternalProducersAndCaptureStartStopListenersInParallel(
   }
 }
 
+static ClientCaptureEvent CreateCaptureFinishedEvent() {
```

Comment:
Maybe move after `CreateCaptureStartedEvent` as it seems more logical.

Reply:
done.